package.json

```diff
@@ -28,7 +28,7 @@
     "promise-pjs": "1.1.3",
     "prop-types": "15.7.2",
     "react-dates": "15.5.3",
-    "rrule": "2.2.0",
+    "rrule": "2.6.2",
```

Comment:
This needs an accompanying change in `yarn.lock`. You can generate it by running `yarn`.

Reply:
@rsimha I tried running `yarn`, but it did not work properly in my local environment.

Am I supposed to set up my own Travis CI environment or run the command locally on my computer?

Is there a way someone else can run yarn to modify my PR?

Reply:
Assigned this to @cvializ